src/Internal/Test/E2E/Wallets.purs

```diff
@@ -169,10 +169,13 @@ namiConfirmAccess extId re =
 
 namiSign :: ExtensionId -> WalletPassword -> RunningE2ETest -> Aff Unit
 namiSign extId wpassword re = do
-  inWalletPage (Pattern $ unExtensionId extId) re signTimeout \nami -> do
-    clickButton "Sign" nami
-    reactSetValue (Selector ":password") wpassword nami
-    clickButton "Confirm" nami
+  inWalletPage (Pattern $ unExtensionId extId) re signTimeout \page -> do
+    clickButton "Sign" page
```

Comment:
We have to Toppokki.pageWaitForSelector for the button too. It does not work on my side.